write_cron.py

```diff
@@ -69,6 +69,11 @@ def get_x(time): return time.split(' ')[1]
         'command': f'{python} {dirpath}/scrapers/crime/scrape_crime.py',
         'comment': 'Scrape crime',
         'time': 'everyday'
+    },
+    {
+        'command': f'{dirpath}/pull_and_run.sh',
+        'comment': 'Pull changes from remote and reset crontab',
+        'time': 'every 7 days'
```

Comment:
Can we change this to every day? Also, silly question, but if we're constantly updating the cron tab, will that screw up the scheduling for programs to run? Can we make sure that nothing is set to run at the same time as this?

Reply:
To address your question, I do not believe that resetting the crontab every update will affect the scheduling of the other jobs (provided the updating doesn't take 12 hours).